msg/sensor_gps.msg

```diff
@@ -1,19 +1,36 @@
-# EKF blended position in WGS84 coordinates.
+# GPS position in WGS84 coordinates.
```

Comment:
Let's fix the things brought up in https://github.com/PX4/Firmware/issues/6275 now.

Reply:
Yes this would probably be the time.